sdk/tests/js/webgl-test-utils.js

```diff
@@ -3137,36 +3137,20 @@ var runSteps = function(steps) {
  *        video is ready.
  */
 function startPlayingAndWaitForVideo(video, callback) {
-  if (video.error) {
-    testFailed('Video failed to load: ' + video.error);
-    return;
-  }
-
-  video.addEventListener(
-      'error', e => { testFailed('Video playback failed: ' + e.message); },
-      true);
-
-  var rvfc = getRequestVidFrameCallback();
-  if (rvfc === undefined) {
-    var timeWatcher = function() {
-      if (video.currentTime > 0) {
-        callback(video);
-      } else {
-        requestAnimFrame.call(window, timeWatcher);
-      }
-    };
-
-    timeWatcher();
-  } else {
-    // Calls video.requestVideoFrameCallback(_ => { callback(video) })
-    rvfc.call(video, _ => { callback(video) });
```

Comment:
This needs to be preserved since the play promise resolving doesn't mean that a frame can be drawn from it yet.

Reply:
I think it probably does mean this? (see comments in main thread)